dataplanes/vpp/pkg/nsmdataplane/nsmdataplane_server.go

```diff
@@ -180,7 +181,8 @@ func (d DataplaneServer) MonitorMechanisms(empty *common.Empty, updateSrv datapl
 			d.remoteMechanisms = update.remoteMechanisms
 			d.localMechanisms = update.localMechanisms
 			if err := updateSrv.Send(&dataplaneapi.MechanismUpdate{
-				RemoteMechanisms: update.remoteMechanism,
+				RemoteMechanisms: update.remoteMechanisms,
+				LocalMechanisms:  update.localMechanisms,
```

Comment:
@mestery #351 lacks this LocalMechanism line.

Reply:
Should this line be added to #351 and merged there?

Reply:
@mestery I'm fine either way... but I think @lobkovilya may be gone for the day... and master is broken now :(